### DISCUSSION TITLE
Tailwind style gone on my live GitHub page  using next.js

### DISCUSSION THREAD

**Discussion by AfamsVal** [tailwindlabs/tailwindcss#5079] [Help]
Why is my Tailwind style not showing on my live Github page but showing on my local computer using next.js

**Comment by mohitsinghs** [comment]
Perhaps, this is because Github Pages considers your build output as Jekyll source causing `_next` folder to be removed.
Try adding an empty `.nojekyll` file in your public folder.

**Reply by AfamsVal** [reply]
Ok, let me try it out now
